config/environments/production.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Specify environment specific hostname and protocol
-  config.hostname = ENV["HOSTNAME"]
+  config.hostname = "#{ENV["HOSTNAME"]}/api"
```

Comment:
instead of HOSTNAME should we maybe call it URL_BASE? or something... eventually it will only be a HOSTNAME, but currently it's not **just** a HOSTNAME